### PR TITLE
replace NULL with nullptr in opencl.hpp file

### PR DESCRIPTION
**Pull request by pwisniewskimobica**
All the NULL keywords were changed to nullptr

### CODE REVIEW

File: tests/test_openclhpp.cpp [hunk 1]
```diff
@@ -153,13 +153,13 @@ static cl_int clGetPlatformInfo_version(
 {
     size_t bytes = strlen(version) + 1;
 
-    TEST_ASSERT_NOT_NULL(id);
+    TEST_ASSERT_NOT_nullptr(id);
```
Comment (bashbaug) [inline tests/test_openclhpp.cpp:156]: I think your search-and-replace was a little too aggressive :smiley:.

These should all remain `TEST_ASSERT_NOT_NULL`.

**Review by bashbaug — CHANGES_REQUESTED** [review]
Mostly LGTM but found one issue that needs to be fixed.

Also needs a trivial rebase.

Thanks!

File: tests/test_openclhpp.cpp (hunk 1 above)
Comment (pwisniewskimobica) [reply tests/test_openclhpp.cpp]: Thanks for this!
Just fixed

**Review by keryell — APPROVED** [review]
Thanks for cleaning 12 years of antiques. :-)

**Comment by bashbaug** [comment]
Looks like something went wrong in the rebase and there are CI failures now - please take a look.

**Comment by pwisniewskimobica** [comment]
Rebased again. thanks

**Review by bashbaug — CHANGES_REQUESTED** [review]

**Review by bashbaug — APPROVED** [review]
Thanks, all good now!

**Pull request merged**